build/package/Installer.MSI.targets

```diff
@@ -146,16 +147,20 @@
     <Target Name="GenerateSdkMSBuildExtensionsNupkg"
             DependsOnTargets="Init;Layout;MsiTargetsSetupInputOutputs;GenerateSdkBundle"
             Condition=" '$(OS)' == 'Windows_NT'"
-            Inputs="$(SdkResolverOutputDirectory)/**/*;
+            Inputs="$(MSBuildExtensionsLayoutDirectory)/**/*;
```

Comment:
What about the resolver? How will its nupkg be generated? It does not seem like it will be part of MSBuildExtensionsLayoutDirectory.